app/src/main/java/org/fossasia/openevent/activities/TracksActivity.java

```diff
@@ -50,17 +59,43 @@ protected void onCreate(Bundle savedInstanceState) {
         recyclerView.setItemAnimator(new DefaultItemAnimator());
     }
 
+
+    private void loadImage() {
+        try {
+            DbSingleton dbSingleton = DbSingleton.getInstance();
+            current = dbSingleton.getTrackbyName(track);
+            Log.d("trying", "picasso");
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+        ImageView backdrop1 = (ImageView) findViewById(R.id.backdrop);
+        if (current.getImage().length() != 0) {
+            Picasso.with(getApplicationContext()).load(current.getImage()).into(backdrop1);
+
+        }
+    }
+
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case android.R.id.home:
                 finish();
                 return true;
+            case R.id.share_tracks:
+                //TODO: Add the real webapp links here
+                Intent intent = new Intent();
```

Comment:
I did ask @championswimmer to discuss about this on gitter, so that we can add a webapp URL, which can be kept constant for all the webapps, that are deployed but I guess he forgot to answer or something.